packages/components/src/components/tabs/_tabs.scss

```diff
@@ -402,6 +402,10 @@
   //-----------------------------
   .#{$prefix}--tab-content {
     padding: $carbon--spacing-05;
+
+    &:focus {
```

Comment:
@asudoh would it be possible to rewrite this to avoid nested styles?

Reply:
I think it's fine for pseudo selectors to be nested since we are doing that throughout this component and most other components

Reply:
@tw15egan it's definitely a pattern we've been using, I think the breakdown is that it makes it difficult to find a specific selector as a maintainer. In other words, if you're looking for `--tab-content:focus` it's not necessarily clear where that is in the source code. When looking for this selector in a nested codebase, you'd have to search around for`--tab-content` and find `:focus` defined underneath it. This can become more challenging if `--tab-content` is used multiple places and you're not sure what result the nested pseudo-selector is located under. 

As a result, I think we'll want to prefer flattening things out with new additions 😕